src/main/java/com/icthh/xm/uaa/web/rest/UserResource.java

```diff
@@ -149,7 +149,7 @@ public ResponseEntity createUser(@Valid @RequestBody UserDTO user) throws URISyn
      */
     @PutMapping("/users/{userKey}/block")
     @Timed
-    @PreAuthorize("hasPermission('USER.BLOCK')")
+    @PreAuthorize("hasPermission(null, 'USER.BLOCK')")
```

Comment:
why null is passed here? 

maybe it is better to pass userKey value instead? Like that:
```
@PreAuthorize("hasPermission({'userKey': #userKey}, 'USER.UNBLOCK')")
``` 
@sergeysenja1992 what do you think?

Reply:
hm, we can do that, but I do not believe that we will use such permission expression. And as I see - all general operations, where we have only user key at input params we use such pre authorize conditions (just name of the action)

Reply:
for me userKey useless in this case
i can't imagine situation when we need hardcode userKey in spel

my resolution: no metter